x-pack/plugins/security_solution/public/common/components/page/index.tsx

```diff
@@ -50,9 +50,9 @@ export const AppGlobalStyle = createGlobalStyle<{ theme: { eui: { euiColorPrimar
   }
 
   /* hide open popovers when a modal is being displayed to prevent them from covering the modal */
-  body.euiBody-hasOverlayMask .euiPopover__panel-isOpen {
-    visibility: hidden !important;
-  }
+  // body.euiBody-hasOverlayMask .euiPopover__panel-isOpen {
+  //   visibility: hidden !important;
+  // }
```

Comment:
Do you want checked in commented out CSS here?

Reply:
No, my mistake. Gonna rebase. Thanks for catching this!